src/full/Agda/Syntax/Abstract.hs

```diff
@@ -15,7 +15,10 @@ import Prelude hiding (null)
 import Data.Bifunctor
 import qualified Data.Foldable as Fold
 import Data.Function (on)
+import Data.List.NonEmpty (NonEmpty)
```

Comment:
There is already an import of `Agda.Utils.List1` which is a wrapper around `Data.List.NonEmpty` and defines type constructor `List1` for non-empty lists.